docs/.vitepress/theme/index.ts

```diff
@@ -1,11 +1,18 @@
 import DefaultTheme from 'vitepress/theme-without-fonts'
 import './custom.css'
-import { Theme } from 'vitepress';
+import { Theme, useData, useRoute } from 'vitepress';
 import VersionSwitcher from "../components/VersionSwitcher.vue";
+import codeblocksFold from 'vitepress-plugin-codeblocks-fold'; // import method
+import 'vitepress-plugin-codeblocks-fold/style/index.css'; // import style
 
 export default {
     extends: DefaultTheme,
     enhanceApp({ app }) {
         app.component('VersionSwitcher', VersionSwitcher)
+    },
+    setup() {
+        const { frontmatter } = useData();
+        const route = useRoute();
+        codeblocksFold({ route, frontmatter }, true, 250);
```

Comment:
```suggestion
        codeblocksFold({ route, frontmatter }, false, 250);
```

I'd rather we use `false` here, and fold codeblocks manually that we want folded, via the frontmatter:
https://github.com/T-miracle/vitepress-plugin-codeblocks-fold?tab=readme-ov-file#extended-use

Reply:
ok